<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Store Inventory Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lstProducts">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copies:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProductName">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblQuantity">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>160</y>
      <width>250</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>610</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove Selected Product</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QFrame" name="SideBarFrmae">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>250</width>
      <height>609</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(223, 233, 239);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>490</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
box-sizing: border-box;
border-radius: 14px;
color: rgb(0, 0, 0);
background-color: rgb(193, 148, 139);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: rgb(193, 148, 139);
box-sizing: border-box;
border-radius: 14px;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QFrame" name="MemberFrame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>211</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="MemberTitle_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(246, 240, 237);
font: 75 9pt &quot;Yu Gothic UI&quot;;
background-color: rgb(42, 83, 106);</string>
        </property>
        <property name="text">
         <string>M E M B E R S</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="MemberBackground_2">
        <property name="styleSheet">
         <string notr="true">background: white;
background-color: rgb(249, 244, 242);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="pushButton_17">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>191</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Yu Gothic UI&quot;;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);

</string>
         </property>
         <property name="text">
          <string>Add Member</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_18">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>191</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Yu Gothic UI&quot;;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Edit Member Details</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_19">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>191</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Yu Gothic UI&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="LibraryFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>211</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="LibraryTitle_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(40, 83, 107, 1);
font: 75 9pt &quot;Yu Gothic UI&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>L I B R A R Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="LibraryBackground_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 244, 242);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="pushButton_20">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>191</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Yu Gothic UI&quot;;
color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>Add Book</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_21">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>191</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Yu Gothic UI&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Update Book</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_22">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>191</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Yu Gothic UI&quot;;
color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>Due Dates</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_23">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>191</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Yu Gothic UI&quot;;
color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>Reserves</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="searchFrame">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #000000;
border-radius: 14px;
background-color: rgb(246, 240, 237);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>531</width>
       <height>21</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string>Type here</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAuthor">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dewey:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>360</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>390</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ID">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>350</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDewey">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuNewProduct"/>
    <addaction name="menuEditSelectedProduct"/>
    <addaction name="menuSaveProducts"/>
    <addaction name="menuLoadProducts"/>
    <addaction name="menuExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuNewProduct">
   <property name="text">
    <string>Add Book</string>
   </property>
  </action>
  <action name="menuEditSelectedProduct">
   <property name="text">
    <string>Update Book</string>
   </property>
  </action>
  <action name="menuSaveProducts">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="menuLoadProducts">
   <property name="text">
    <string>Load Stock</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
